metadata.yaml

```diff
@@ -9,6 +9,9 @@ releaseSeries:
 - major: 1
   minor: 1
   contract: v1beta1
+- major: 0
+  minor: 6
+  contract: v1beta1
```

Comment:
I'm curious about how this version relates to upstream ClusterAPI versions.

I'm not getting a lot of help from https://cluster-api.sigs.k8s.io/clusterctl/provider-contract.html#metadata-yaml

Reply:
The new entry for 0.6 looks fine and seems to correlate with the latest upstream releases.

On a slightly unrelated note, the v1.1 entry above seems like a mistake, since the current versioning is v0.x.
If clusterctl-settings.json was updated, then I think this entry could be removed...  To keep dev versions higher than released versions, clusterctl-settings.json could be updated to use a `nextVersion` of `v0.6.99` until the point that a new minor (or major version is released), in which case `nextVersion` could be updated along with the other requirements for a version bump.

Reply:
@displague the lack of helpfulness of the metadata documentation would be great feedback for upstream

Reply:
So I went with 1.1.99 as that's what the upstream e2e modules want for their nextVersion. But I'll try your advice about overriding it in clusterctl-settings.json and see if that works!

Reply:
For my future reference, the nextVersion in clusterctl-settings was always what was controlling this. I must have been scared to changed it originally, now I know it's safe and the right way to set this.